resources/sass/app.scss

```diff
@@ -262,70 +260,71 @@ dl.import-meta .download-csv {
+    background-color: $wikit-Button-normal-neutral-background-color;
+    transition-duration: $wikit-Button-transition-duration;
 
     &:hover {
         text-decoration: none;
-        color: $color-modifier-lighten-base-10;
-        border-color: $color-base-50;
-        background-color: $color-base-100;
+        color: $wikit-Button-normal-neutral-hover-color;
+        border-color: $wikit-Button-normal-neutral-hover-border-color;
+        background-color: $wikit-Button-normal-neutral-hover-background-color;
         transition-property: background-color, border-color;
     }
 
     &:active {
-        color: $color-base-0;
-        border-color: $color-base-30;
-        background-color: $color-base-80;
+        color: $wikit-Button-normal-neutral-active-color;
+        border-color: $wikit-Button-normal-neutral-active-border-color;
+        background-color: $color-base-80; // TODO not $wikit-Button-normal-neutral-active-background-color?
         transition-property: background-color, border-color;
     }
 
     &:focus {
-        color: $color-base-10;
-        background-color: $color-base-90;
-        border-color: $color-accent-50;
-        box-shadow: $color-accent-50 0 0 0 1px inset;
+        color: $wikit-Button-normal-neutral-focus-color;
+        background-color: $wikit-Button-normal-neutral-focus-background-color;
+        border-color: $wikit-Button-normal-neutral-focus-border-color;
+        box-shadow: $wikit-Button-normal-neutral-focus-box-shadow;
         transition-property: box-shadow;
     }
 
     &.progressive {
-        color: $color-base-100;
-        background-color: $color-accent-50;
-        border-color: $color-accent-50;
+        color: $wikit-Button-primary-color;
+        background-color: $wikit-Button-primary-progressive-background-color;
+        border-color: $wikit-Button-primary-progressive-border-color;
 
         &:visited {
             color: white;
         }
 
         &:hover {
-            background-color: $color-modifier-lighten-accent-50;
-            border-color: $color-modifier-lighten-accent-50;
+            background-color: $wikit-Button-primary-progressive-hover-background-color;
+            border-color: $wikit-Button-primary-progressive-hover-border-color;
             transition-property: background-color, border-color;
         }
         &:active {
-            background-color: $color-accent-30;
-            border-color: $color-accent-30;
+            background-color: $wikit-Button-primary-progressive-active-background-color;
+            border-color: $wikit-Button-primary-progressive-active-border-color;
             transition-property: background-color, border-color;
         }
         &:focus {
-            box-shadow: $color-base-100 0 0 0 1px inset;
-            border-color: $color-accent-50;
-            background-color: $color-accent-50;
+            box-shadow: $color-base-100 0 0 0 1px inset; // TODO not $wikit-Button-primary-progressive-focus-box-shadow?
```

Comment:
Yes, this token exists: https://github.com/wmde/wikit/blob/master/tokens/properties/components/Button.json#L288

Reply:
Done in bba763be328c5e94d3eda0be8acab4717fca42cc.